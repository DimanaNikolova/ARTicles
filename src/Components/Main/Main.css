.Main{
    column-count: 3;
}