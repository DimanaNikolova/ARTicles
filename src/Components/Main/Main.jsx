import React from 'react'
import './Main.css'
import Header from '..//Header/Header'
import Footer from '../Footer/Footer'


const Main = (props) => {  
        return (
                <div className="App">
                    <Header />
                        <div className="Main">
                            {props.children}
                        </div>                  
                    <Footer />
                </div>
            
        )
}

export default Main