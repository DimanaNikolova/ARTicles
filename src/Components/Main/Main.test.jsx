import React from 'react'
import renderer from 'react-test-renderer'
import Main from './Main'
import TestingEnvironment from '../../utils/router-test'

jest.mock('../Header/Header.jsx', () => 'Header');
jest.mock('../Footer/Footer.jsx', () => 'Footer');

describe('Main Component', () => {
  it('should render main component', () => {
    
    const tree = renderer.create(
      <TestingEnvironment value={{
        user: true
      }}>
        <Main />
      </TestingEnvironment>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})