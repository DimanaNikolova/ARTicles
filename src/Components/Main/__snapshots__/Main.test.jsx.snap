// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Component should render main component 1`] = `
<div
  className="App"
>
  <Header />
  <Footer />
</div>
`;
